### DISCUSSION TITLE
Questions Zoom Meeting 21.02.2024

### DISCUSSION THREAD

**Discussion by Happinessjump** [cest-sources/cestineers#5] [Q&A]
Questions from the Zoom Chat

**Comment by Happinessjump** [comment]
Is the protocol APTw only or can you adjust B1, offsets, etc?

**Reply by mzaiss** [reply]
For all sequences inlcluding Siemens WIPS this can be altered.
The online recon of Siemens will only work well for certain offset lists though.

**Reply by ghost** [reply]
in the ini file of the WIP you can specify what range and what center frequency should be used for the asymmetry. Also check the appl guide

**Comment by Happinessjump** [comment]
Is there a time line for when XA versions will be available?

**Comment by Happinessjump** [comment]
Do these sequences run in regular user mode?
-Do we need to be in advanced user to run these sequences?

**Reply by mzaiss** [reply]
Yes, normal user mode..

**Comment by Happinessjump** [comment]
Is CEST going to be a Siemens product?

**Reply by mzaiss** [reply]
no reply.

**Comment by Happinessjump** [comment]
CS = 8 and 10, how good are they?

**Reply by mzaiss** [reply]
ok, if you use enough offsets and variable patterns.

**Comment by Happinessjump** [comment]
When should I use the WIP and when should I prefer the C2P by Moritz?

**Reply by mzaiss** [reply]
More resolution and coverage: WIP
More advanced preparations: C2P
But the C2P has also more conservative and decent SNR.

**Comment by Happinessjump** [comment]
Thanks for the presentation Patrick: Does the 816C use temporal incoherence already?

**Reply by Happinessjump** [reply]
@author: yes you can select temporal incoherence on the special card. And then you have to activate the second regularization parameter to activate the TV regularization. If it is not selectable then there may be some UI problem...

**Comment by Happinessjump** [comment]
Do you not correct gibbs ringing artifacts?

**Reply by mzaiss** [reply]
no

**Comment by Happinessjump** [comment]
A lot of changes on FS APTw can be seen, including not fluid brain areas. It seems that it has been suppressed too much. What do you think?

**Reply by mzaiss** [reply]
Discussed in detail here: https://onlinelibrary.wiley.com/doi/10.1002/mrm.29915

**Comment by Happinessjump** [comment]
We installed WIP 816C in a Vida and in the user guide in references ini files for the CEST parameters. I didn’t see them mentioned here. Do use ini files still?

**Reply by mzaiss** [reply]
Siemens uses ini files.
C2P use typically settings on the special card.

**Comment by Happinessjump** [comment]
Do you have any suggestions on preparation module for ex-vivo glyco-CEST at bruker 9.4 T?

**Reply by mzaiss** [reply]
https://pubmed.ncbi.nlm.nih.gov/31313865/

**Comment by Happinessjump** [comment]
What is the current spatial detection limit for Siemens? Thanks

**Reply by mzaiss** [reply]
1.8 mm isotropic whole brain with the CS recon

**Comment by Happinessjump** [comment]
Is there an intention to integrate/enable pulseq-CEST .seq format in Siemens' future WIP? Does the new WIP 816C has a known pulseq-CEST .seq file?

**Reply by ghost** [reply]
it is somewhere on the list, but currently not a top priority, plus I would first like to double check with Maxim :)

**Comment by Happinessjump** [comment]
A quick question for pulseq, how did you choose the pool size fraction in yaml file for phantom study (l-arginin in pulses-cest-library for example ?

**Reply by mzaiss** [reply]
for phantoms you can calculate it by number of protons*[larginin]/[water protons]  

[water protons] =111M

but I am not sure about the proton number.

**Comment by Happinessjump** [comment]
Did anyone possibly try out to derive a B0 map from the shimming measurement raw data?

**Reply by ghost** [reply]
I would not do that, since the protocol for estimating the shim coefficients does not have a very high resolution. So I would rather use your own protocol. You can modify the protocol for B0 mapping in the WIP, see appl guide.

**Comment by Happinessjump** [comment]
I had a question about the C2P protocol/guidelines on XA30/50… there is a comment about automatic adjustments… do we still need to set the adjust volume, do manual shimming, etc, or is this automated now?

**Reply by mzaiss** [reply]
we use standard shim and auto align at 3T.
manual shim at 7T.

**Comment by Happinessjump** [comment]
What is the highest B1 field in the Siemens? Thanks again.

**Reply by Happinessjump** [reply]
Are you referring to 3T or 7T CEST saturation? Because they use different SAR estimations and limits and are therefore different. It also depends on the Duty cycle.

**Reply by Happinessjump** [reply]
Please incude both, if possible

**Reply by Happinessjump** [reply]
https://analyticalsciencejournals.onlinelibrary.wiley.com/doi/10.1002/nbm.5096

**Reply by Happinessjump** [reply]
Thank you so much for the paper and explanation

**Comment by Happinessjump** [comment]
What about FASTESTMAP for better and quick shimming?

**Reply by mzaiss** [reply]
yes all possible, better shim, better CEST.
